### PR TITLE
libjpeg: new package at v9d

### PR DESCRIPTION
**Pull request by TAAPArthur**
Please fill out the template.

Read: https://kiss.armaanb.net/wiki/kiss/style-guide

---

## Description of package
A widely used free library for JPEG image compression

## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/libjpeg/version
/var/db/kiss/installed/libjpeg/sources
/var/db/kiss/installed/libjpeg/manifest
/var/db/kiss/installed/libjpeg/checksums
/var/db/kiss/installed/libjpeg/build
/var/db/kiss/installed/libjpeg/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/local/share/man/man1/wrjpgcom.1
/usr/local/share/man/man1/rdjpgcom.1
/usr/local/share/man/man1/jpegtran.1
/usr/local/share/man/man1/djpeg.1
/usr/local/share/man/man1/cjpeg.1
/usr/local/share/man/man1/
/usr/local/share/man/
/usr/local/share/
/usr/local/lib/pkgconfig/libjpeg.pc
/usr/local/lib/pkgconfig/
/usr/local/lib/libjpeg.so.9.4.0
/usr/local/lib/libjpeg.so.9
/usr/local/lib/libjpeg.so
/usr/local/lib/libjpeg.la
/usr/local/lib/libjpeg.a
/usr/local/lib/
/usr/local/include/jpeglib.h
/usr/local/include/jmorecfg.h
/usr/local/include/jerror.h
/usr/local/include/jconfig.h
/usr/local/include/
/usr/local/bin/wrjpgcom
/usr/local/bin/rdjpgcom
/usr/local/bin/jpegtran
/usr/local/bin/djpeg
/usr/local/bin/cjpeg
/usr/local/bin/
/usr/local/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/libjpeg/build [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+./configure
```
Comment (git-bruh) [inline community/libjpeg/build:3]: it should be:
```
./configure \
    --prefix=/usr
```
Check the style guide

**Review by git-bruh — CHANGES_REQUESTED** [review]

**Comment by dilyn-corner** [comment]
Is there a good reason to have `libjpeg`? We already have `libjpeg-turbo`

**Comment by TAAPArthur** [comment]
@dilyn-corner My rationale for using libjpeg instead of libjpeg-turbo was solely to avoid the cmake make dependency. Personally am not too found of cmake and didn't feel like attempting to get it working with tcc.

**Comment by dilyn-corner** [comment]
Avoiding `cmake` sounds like a reasonable reason to me. Users should just be aware that `libjpeg-turbo` will remain the default in repo/extra and if they wish to avoid that they will have to do slightly more work :)

**Pull request merged**